### ISSUE TITLE
Fields in translation are not update

### ISSUE THREAD

**Issue by jbitsch** [directus/api#1137]
# Bug Report

## Steps to Reproduce

1. git clone `https://github.com/directus/api .` and setup scheme based on src/scheme.sql
2. `composer install`
3. Copi version of api.php
4. Create 3 new collections called `test_main` `test_translation` and `language`
5. The `language` collection contains the field `name` and a primary key `id`. 
6. Insert one language entry in the `language` collection
7. Create the two fields `test_text` and `language_code` in the `test_translation` collection
8. Create a translation field in the `test_main` which links to the  `test_translation` collection:
![image](https://user-images.githubusercontent.com/1073322/61548961-32077a80-aa4f-11e9-85ed-f130218778f5.png)

9. Create and entry in the `test_main` collection and see that i saves the translation correctly. 
10. Try to update the text in the translation.

## Expected Behavior
When creating an item the fields in the translation should be saved. 
When updating an item the fields in the translation should be updated.

## Actual Behavior
When creating an item the fields in the translation should be saved.  (It does)
When updating an item the fields in the translation are NOT being updated.

## Technical Details

- Device: Desktop
- OS: MacOS + Google App Engine
- Web Server: Apache 2.4.37
- PHP Version:  7.2.0
- Database:  MySQL 8.0.12
- Install Method:  cloned `master` branch
- Use the master branch for both the app and the api

**Comment by itsmerhp** [comment]
@jbitsch 
Have you created a column in `test_text`(eg. test_id) collection to relate `test_text` collection with `test_main` collection?

Please check below screenshots for reference : 
![Settings   Editing Item  Test Translation](https://user-images.githubusercontent.com/49250107/61582776-c1468800-ab4c-11e9-9884-265524bbdb6a.png)
![Settings   Editing Item  Test Main](https://user-images.githubusercontent.com/49250107/61582777-c1468800-ab4c-11e9-9ab8-e3fd43703fe8.png)

Please follow this link for the setup of translation feature: [Multilingual Content Setup in Directus (i18n)](https://medium.com/directus/multilingual-content-setup-in-directus-i18n-4f243f72e554)

**Comment by jbitsch** [comment]
@itsmerhp  yes i have, and when creating an item with a translation, the translation is also created correctly, and when displaying the created entry, it shows the created translation. But when trying to update the translation field, the value is not saved.

**Comment by jbitsch** [comment]
A site not to this is that we are migrating from Directus v6 to v7, where this worked, and we have not changed the setup.

**Comment by itsmerhp** [comment]
@jbitsch Can you please share SQL dump?

**Comment by jbitsch** [comment]
@itsmerhp  Here is a link: https://drive.google.com/open?id=1WS_U-4tRX7vopjur-u3LsODvqZCwOXLP

I do not know if it is related to this where in the overview the id is NaN
![image](https://user-images.githubusercontent.com/1073322/61583040-da423f80-ab32-11e9-888c-f1a06d969746.png)

But when viewing the entry is is the correct data from the database = 4:
![image](https://user-images.githubusercontent.com/1073322/61583044-e8905b80-ab32-11e9-95cf-bbc740a756e2.png)

When viewing the main entry the field is not filled:
![image](https://user-images.githubusercontent.com/1073322/61583049-0bbb0b00-ab33-11e9-8fa1-2437ff98dcaa.png)

**Comment by itsmerhp** [comment]
@jbitsch Thank you for sharing SQL dump.
@rijkvanzanten While updating translation record from the main table in APP, in API request params, updated values are not passing(Old values are passing), please check it in [this video](https://drive.google.com/file/d/11jnquMVpKn76LLgv5MPLlIl7oud1MYm0/view)

.

**Comment by benhaynes** [comment]
Hey @itsmerhp — should we open a ticket on the App so this is tracked?

**Comment by itsmerhp** [comment]
@benhaynes 
Yes right, this ticket should be moved to the APP.
